tox_poetry_installer/hooks.py

```diff
@@ -219,24 +133,28 @@ def tox_testenv_install_deps(venv: ToxVirtualEnv, action: ToxAction) -> Optional
         )
 
         env_deps = utilities.find_additional_deps(
-            packages, virtualenv, poetry, venv.envconfig.locked_deps
+            packages, virtualenv, poetry, tox_env.conf["locked_deps"]
         )
 
         logger.info(
             f"Identified {len(env_deps)} environment dependencies to install to env"
         )
 
         install_project_deps = (
-            venv.envconfig.install_project_deps
-            if venv.envconfig.install_project_deps is not None
-            else (
-                not venv.envconfig.skip_install and not venv.envconfig.config.skipsdist
-            )
+            tox_env.conf["install_project_deps"]
+            if tox_env.conf["install_project_deps"] is not None
+            else (not tox_env.conf["skip_install"] and not tox_env.core["no_package"])
         )
 
+        # extras are not set in a testenv if skip_install=true
+        try:
+            extras = tox_env.conf["extras"]
+        except KeyError:
+            extras = []
```

Comment:
Is `tox_env.conf` a regular dictionary? I.e. does it work to use `dict.get()` here?
```python
extras = tox_env.conf.get("extras", [])
```

Reply:
Unfortunately that does not seem to be the case. Only a `load` method is defined in the base class `ConfigSet`, which is the one called in `__getitem__`. That method does not support providing a default value via parameters.